packages/suite/src/components/suite/Notifications/components/NotificationGroup/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { NotificationEntry } from '@suite-reducers/notificationReducer';
+import { AppState } from '@suite-types';
+import { variables, P } from '@trezor/components';
+import { Translation } from '@suite-components';
+import React from 'react';
+import styled from 'styled-components';
+import NotificationList from '../NotificationList';
+
+const SectionHeadline = styled.div`
+    margin-top: 14px;
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    text-transform: uppercase;
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    height: 16px;
+    line-height: 1.33;
+    letter-spacing: 0.2px;
```

Comment:
Does this work nicely on both Retina and non-Retina displays?

Reply:
I don't know how to test this properly (I have a Linux machine and I simulated different devices in Chrome Dev Tools - which looks ok).

Reply:
didn't notice anything weird on my mac :)